OSDOCS-5976: MicroShift storage section tweaks (#63439)
openshift/openshift-docs: modules/storage-persistent-storage-pv.adoc

|===

> You can view the name of the PVC when it is bound to the PV by running the following command:

Reviewer: ```suggestion
You can view the name of the PVC that is bound to the PV by running the following command:
```

You can get the PVC name other ways. This was about getting the PVC associated with a PV.

Reviewer 2: @ShaunaDiaz I'm OK with this change,  just suggested to update the command from "oc get pv <pv-claim>" to "oc get pv <pv-name>"
